presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -121,6 +121,7 @@
 import com.facebook.presto.sql.tree.QualifiedName;
 import com.facebook.presto.type.BigintOperators;
 import com.facebook.presto.type.BooleanOperators;
+import com.facebook.presto.type.CharOperators;
```

Comment:
the commit message for this commit is misleading.
It adds all the equality/non-equality operators for char. Not only for chars of equal length. Maybe split it into two or change commit message.